NorthstarDedicatedTest/sigscanning.cpp

```diff
@@ -5,7 +5,7 @@
 // note: sigscanning is only really intended to be used for resolving stuff like shared function definitions
 // we mostly use raw function addresses for stuff
 
-size_t GetDLLLength(HMODULE moduleHandle)
+size_t GetDLLength(HMODULE moduleHandle)
```

Comment:
Should actually be DLLLength, 3 Ls cause 2 in DLL and 1 in Length

Reply:
Fixed, oops